source/blog/2018-11-16-the-ember-times-issue-73.md

```diff
@@ -44,8 +44,16 @@ Have you heard about [Hooks in React](https://reactjs.org/docs/hooks-intro.html)
 
 ---
 
-## [SECTION TITLE](#section-url)
+## [Videos From EmberFest Are Out!](https://www.youtube.com/playlist?list=PLN4SpDLOSVkSB9034lDNdP1JoNBGssax9)
```

Comment:
Maybe add an Emoji here?
Some ideas:
https://emojipedia.org/video-camera/
https://emojipedia.org/television/
https://emojipedia.org/dvd/

Reply:
I just saw these suggestions! I suggested video at the top so added party/celebrate in here for now (feel free to change - sounds like more edits are coming)